README.md

```
@@ -132,42 +184,15 @@ https://www.u-hyu.site/
-
-#### 🧐 기타 고려 사항
-
-- **관심 브랜드를 `users` 테이블에 직접 저장하지 않음**  
-  → 추천 전용 도메인으로 분리하여 관리하는 것이 유지보수 및 확장성 측면에서 바람직
-- 추천 로직은  
-  - 초기 추천: `recommendation_base_data` 사용  
-
----
+| 사진 | 이름 | GitHub |
+|------|------|--------|
+| <img src="https://avatars.githubusercontent.com/u/127932430?v=4" width="80"> | 👑임동준 | [djlim00](https://github.com/djlim00) |
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)